Mathlib/Analysis/Calculus/ContDiff/FTaylorSeries.lean

```diff
@@ -622,6 +630,30 @@ theorem HasFTaylorSeriesUpToOn.eq_iteratedFDerivWithin_of_uniqueDiffOn
 alias HasFTaylorSeriesUpToOn.eq_ftaylor_series_of_uniqueDiffOn :=
   HasFTaylorSeriesUpToOn.eq_iteratedFDerivWithin_of_uniqueDiffOn
 
+/-- The iterated derivative commutes with shifting the function by a constant on the left. -/
+lemma iteratedFDerivWithin_comp_add_left (n : ℕ) (a : E) :
+    iteratedFDerivWithin 𝕜 n (fun z ↦ f (a + z)) s =
+      fun x ↦ iteratedFDerivWithin 𝕜 n f (a +ᵥ s) (a + x) := by
```

Comment:
It strikes me as a bit odd that these statements are not applied further, but fine if this is more useful. (Or maybe have both?)

Reply:
Indeed, the unapplied version is useful in the proof, but I agree the applied version is maybe more natural. I've added both.